app.js

```diff
@@ -0,0 +1,74 @@
+const addMovieBtn = document.getElementById("add-movie-btn");
+const searchBtn = document.getElementById("search-btn");
+
+const movies = [];
+
+const filtermovieHandler = () => {
+  const filteredElement = document.getElementById("filter-title").value;
+  renderMovieElement(filteredElement);
+};
+
+const renderMovieElement = (filter = "") => {
+  const movieList = document.getElementById("movie-list");
+
+  if (movies.length === 0) {
+    movieList.classList.remove("visible");
+    return;
+  } else {
+    movieList.classList.add("visible");
+  }
+
+  movieList.innerHTML = "";
+
+  const filtermovieElement = !filter
+    ? movies
+    : movies.filter((movie) => movie.info.title.includes(filter));
+
+  filtermovieElement.forEach((movie) => {
+    const movieLi = document.createElement("li");
+    const { info, ...otherkeys } = movie;
+    console.log(otherkeys);
+    // const { title: movieTitle } = info;
+    // const { getFormattedTitle } = movie;
+    let editedtext = movie.getFormattedTitle() + " - ";
+    for (const otherKeys in info) {
+      if (otherKeys !== "title") {
+        editedtext = editedtext + `${otherKeys} : ${info[otherKeys]}`;
```

Comment:
+= ?

Reply:
Thanks for reminding shortcuts @faran4engg . i've made this change